src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
I wonder how we should formulate comment like this - we know problem exist in Chrome (in some version) and IE9 but it might also exist (and probably does like in Safari) in other browsers?